src/main/webapp/app/pages/variantRecommendation/FormatPercentage.tsx

```diff
@@ -0,0 +1,19 @@
+export function formatPercentage(value: number) {
+  if (value < 1e-4 && value > 0) {
+    let formatted = (value * 1000).toFixed(2);
```

Comment:
Why do you need to multiply by 1000?

Reply:
Because some numbers are very small, they can only be expressed in ‰ instead of %.

Reply:
Let's keep it as a percentage and not be per thousand. I think it'll confuse people if we change the units when they're reading through the rows.